lib/jsdom/living/validity/ValidityState-impl.js

```diff
@@ -0,0 +1,66 @@
+"use strict";
```

Comment:
You could name this file `lib/jsdom/living/form-control/ValidityState-impl.js` to make the directory a bit more generic.